windows/src/developer/inst/copydev.in

```diff
@@ -1,7 +1,7 @@
 !include ..\..\Defines.mak
 
 ##
-## In this file, $VERSION and $RELEASE will be replaced by mkver. These are not
+## In this file, $Version, $VersionWin and $VersionRelease will be replaced by mkver. These are not
```

Comment:
Is $VersionWithTag also replaced by mkver?

Reply:
yes